src/components/general/mediaTypes/ImageViewer.tsx

```diff
@@ -81,7 +81,7 @@ const ImageViewer = (props: Props) => {
             if (viewerRef.current) {
                 viewerRef.current.open({
                     type: 'image',
-                    url: mediaUrl
+                    url: 'https://images.pexels.com/photos/14078434/pexels-photo-14078434.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1'
```

Comment:
Is the hardcoded link correct?

Reply:
Was for testing, reverted to dynamic mediaUrl